2gp-demo/force-app/main/default/classes/LicensingUtil.cls

```diff
@@ -1,49 +1,32 @@
 public with sharing class LicensingUtil {
 	
- 	private static String currentNamespace = getNamespace();
-    private static Set<String> packageNamespaces = new Set<String> {'plpdp_mynamespace1', 'IsvlTdxDemo'};
-    
-	
     public static boolean hasAccessToLCP(String customPerm) {
         // TODO: Confirm that the customPerm is not null, exists, and is a Licensed Custom Perm - otherwise throw exception
         
         /*
         System.debug('debug Test.isRunningTest ' + Test.isRunningTest());
         System.debug('debug isDevOrg ' + isDevOrg());
-        System.debug('debug isPackagingOrg ' + isPackagingOrg());
         System.debug('debug userHasPackageLicense ' + userHasPackageLicense());
         System.debug('debug FeatureManagement.checkPermission ' + FeatureManagement.checkPermission(customPerm));
-        System.debug('debug getNamespace ' + getNamespace());
         System.debug('debug getPackageId ' + getPackageId());
         */
         
         if(Test.isRunningTest() ||                           // bypass access check for automated tests
            isDevOrg() ||                                     // bypass access check if within a developer org (without local namespace)
-           isPackagingOrg() ||                               // bypass access check if within your packaging org (with local namespace)
-           userHasPackageLicense() ||         				 // allow access if the user has a managed package license (for hybrid licensing)
+           //userHasPackageLicense() ||         				 // allow access if the user has a managed package license (for hybrid licensing)
```

Comment:
Append to comment: Commented out since this method always returns true in scratch org context. Scratch orgs by default are on a "site-wide" package license